modules/plugins/debugger/nvim-dap/config.nix

```diff
@@ -52,24 +52,31 @@ in {
     })
     (mkIf (cfg.enable && cfg.ui.enable) {
       vim = {
-        startPlugins = ["nvim-dap-ui" "nvim-nio"];
+        startPlugins = ["nvim-nio"];
 
-        pluginRC.nvim-dap-ui = entryAfter ["nvim-dap"] (''
-            local dapui = require("dapui")
-            dapui.setup()
-          ''
-          + optionalString cfg.ui.autoStart ''
+        lazy.plugins.nvim-dap-ui = {
+          package = "nvim-dap-ui";
+          setupModule = "dapui";
+          setupOpts = {};
```

Comment:
Do we really need to specify empty setupOpts? Also why doesn't the module have a setupOpts Nix option
```suggestion
```